### PR TITLE
Update base dependencies

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Removes base dependency from components which are not using it. Bumps base in the site package to the latest major release.

Thanks for contributing to Terra. 
@cerner/terra

### CODE REVIEW

**Comment by alex-bezek** [comment]
@mhemesath does this fit your needs to get nav working with the right base.css file for portal rather than upgrading all our terra dependencies?
https://jira2.cerner.com/browse/PORTALDEV-36251

File: packages/terra-consumer-icon/package.json [hunk 1]
```diff
@@ -33,12 +33,10 @@
   "peerDependencies": {
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
```
Comment (alex-bezek) [inline packages/terra-consumer-icon/package.json]: the comma on this line needs to be removed as well @bjankord . Thanks for making this PR to our repo!

**Comment by mhemesath** [comment]
I think we need to update  every terra dependency because they are still referencing the old version of terra base

**Comment by alex-bezek** [comment]
Looks like @amf9t2 is fixing that issue here https://github.com/cerner/terra-consumer/pull/32

**Comment by bjankord** [comment]
We can close this in favor of using @amf9t2's PR #32

**Pull request closed without merging**